src/functions-reference/array_operations.Rmd

```diff
@@ -179,6 +179,24 @@ The squared Euclidean distance between x and y
 `real` **`squared_distance`**`(row_vector x, row_vector y)`<br>\newline
 The Euclidean distance between x and y
 
+### Quantile
+
+Produces sample quantiles corresponding to the given probabilities.
+The smallest observation corresponds to a probability of 0 and the largest
+to a probability of 1.
+
+<!-- real; quantile; (data real[] xs, data real p); -->
```

Comment:
I think convert all the `xs` to just `x` and all the `ps` to just `p`. There are some inconsistencies here to probly better to just convert them all to one thing.